barretenberg/cpp/pil/avm/kernel.pil

```diff
@@ -2,11 +2,11 @@ include "main.pil";
 include "constants.pil";
 
 namespace kernel(256);
-    pol public(/*idx=*/0) kernel_inputs;
+    pol public kernel_inputs;
```

Comment:
In the fork, public polynomials have been made a first class citizen, so we can call get_in_source_order for them, which means we will not have ordering issues anymore